tests/integration/Cron/DaemonHttpRunnerTest.php

```diff
@@ -68,20 +71,19 @@ function testItStoresErrorMessageAndContinuesExecutionWhenWorkersThrowException(
       'token' => 123,
     ];
 
-    $workers_factory_mock = $this->createWorkersFactoryMock([
-      'createScheduleWorker' => $this->makeEmpty(SimpleWorker::class, [
-        'process' => function () {
+    $run = 0;
+    $cron_worker_runner_mock = $this->make(CronWorkerRunner::class, [
```

Comment:
Is there a better way to create this mock? If we add logic to our tests should we also test it?

Reply:
With Codeception stubs it is not possible to pass callbacks on consecutive calls, only values (https://codeception.com/docs/reference/Stub#public-static-consecutive). Maybe I can mock it with PHPUnit, it won't be much simpler than now I guess.